lib/compression.js

```diff
@@ -8,21 +8,23 @@ const Hoek = require('@hapi/hoek');
 
 
 const internals = {
-    common: ['gzip, deflate', 'deflate, gzip', 'gzip', 'deflate', 'gzip, deflate, br']
+    common: ['gzip, deflate', 'deflate, gzip', 'gzip', 'deflate', 'br', 'gzip, deflate, br']
 };
 
 
 exports = module.exports = internals.Compression = class {
 
     decoders = {
+        br: (options) => Zlib.createBrotliDecompress(options),
         gzip: (options) => Zlib.createGunzip(options),
         deflate: (options) => Zlib.createInflate(options)
     };
 
-    encodings = ['identity', 'gzip', 'deflate'];
+    encodings = ['identity', 'gzip', 'deflate', 'br'];
```

Comment:
This should probably appear before `'gzip'`. Otherwise gzip will be preferred for all practical use-cases.

Reply:
IMO, the order thingy should be addressed with `compression.priority` feature, which I plan to add if this one is lands 😎

Reply:
IMO, the order should stay as is, bcos in current implementation `gzip` used by default when client sends `accept-encoding: 'gzip, deflate, br'`, and changing the order will be a breaking change. Best way to address this is to add `compression.priority` feature to make it possible flexibly change the order of compression algorithms.